README.rst

```diff
@@ -16,7 +16,7 @@ About
 
 ``reana-auth-vomsproxy`` provides a container image for creating and renewing a VOMS proxy certificate. The container image
 includes no additional logic or libraries, just the bare minimum to support the VOMS client.
-The image is configured to support authentication for the four experiments at CERN: ALICE, ATLAS, CMS and LHCb.
+The image is configured to support authentication for the four experiments at CERN: ALICE, ATLAS, CMS and LHCb. It also supports authentication for the ESCAPE scientific cluster. 
```

Comment:
Cosmetics: What about saying; "... authentication for the four experiments at CERN Large Hadron Collider (ALICE, ATLAS, CMS, LHCb) as well as ESCAPE Virtual Organization.

(And since "escape" is a common word hard to google, perhaps add a web link?)

Reply:
Agreed. Link: https://projectescape.eu/